package.json

```diff
@@ -73,7 +73,9 @@
         "onLanguage:python",
         "onLanguage:csharp",
         "onLanguage:yaml",
-        "onCommand:aws.launchConfigForm"
+        "onCommand:aws.launchConfigForm",
+        "onCommand:aws.toggleSamCodeLenses",
```

Comment:
Is the rationale for this command, that it might be a common action? If it's not common it should probably just be an option. Unless we think customers will want to hide the hints, and later show them.

Reply:
Yes, at the very least it'll be more accessible. I don't know how often people toggle any AWS-related settings...that'd be an interesting one to get metrics on.